src/BlazorUI/Bit.BlazorUI/Components/Inputs/TagsInput/BitTagsInput.scss

```
@@ -23,6 +23,11 @@
         .bit-tgi-cnt {
             border-color: $clr-err;
         }
+
+        &.bit-tgi-fcs .bit-tgi-cnt {
+            @include focus-ring($clr-err-focus);
+            border-color: $clr-err-focus;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)